src/main/java/core/basesyntax/figure/FigureSupplier.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax.figure;
+
+import java.util.Random;
+
+public class FigureSupplier {
+
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+
+        int figureType = random.nextInt(5);
+        String color = colorSupplier.getRandomColor();
+        int maxValue = 100;
```

Comment:
The variable 'maxValue' is used as a maximum value for generating random numbers; it should be declared as a constant field in the class.

Reply:
Fix the comment